src/EFCore.PG/Storage/Internal/Mapping/NpgsqlJsonObjectTypeMapping.cs

```diff
@@ -29,7 +30,10 @@ protected NpgsqlJsonObjectTypeMapping(RelationalTypeMappingParameters parameters
         protected override RelationalTypeMapping Clone(RelationalTypeMappingParameters parameters)
             => new NpgsqlJsonObjectTypeMapping(parameters, NpgsqlDbType);
 
+        protected virtual string EscapeSqlLiteral([NotNull] string literal)
```

Comment:
Any reason to be virtual?

Reply:
Not really, this is just the standard way to do it in EF Core, in the theoretical case someone wants to override etc...